File: Segundo Semestre/LAB121/Proyecto/Escultura.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Proyecto
{
    public class Escultura:Arte
    {
        private string Material;
        public string material
        {
            get { return Material; }
            set { Material = value; }
        }
        public void Leer()
        {
            System.Console.WriteLine("Leer genero: ");
            Material = Console.ReadLine();
        }
        public void Mostrar()
        {
            base.Mostrar();
            System.Console.WriteLine(Material);
        }
    }
}

File: antiguoPlan/segundoSemestre/lab121/Proyecto/JavaInterfaz/src/Escultura.java

import java.util.Scanner;

    public class Escultura extends Arte
    {

    public Escultura(String Material, int AñodePublicacion, String NombredelaObra, String Popularidad, String Aceptacion) {
        super(AñodePublicacion, NombredelaObra, Popularidad, Aceptacion);
        this.Material = Material;
    }
        private String Material;
        /*public string material
        {
            get { return Material; }
            set { Material = value; }
        }*/
        Scanner sc = new Scanner(System.in);
        /*
        System.out.println(
        sc.nextLine();
        */
        public void Leer()
        {
            System.out.println("Leer genero: ");
            Material = sc.nextLine();
        }
        public void Mostrar()
        {
            super.Mostrar();
            System.out.println(Material);
        }
    }
